selfdrive/ui/ui.cc

```diff
@@ -112,7 +112,7 @@ static void update_state(UIState *s) {
   }
   if (sm.updated("radarState")) {
     std::optional<cereal::ModelDataV2::XYZTData::Reader> line;
-    if (sm.rcv_frame("modelV2") > 0) {
```

Comment:
we added this if statement before because we encountered  segfalut without it. so I think the new if statement is more logically correct and safe.

Reply:
why is line optional for update_leads? should we just skip the call if there is no modelV2 yet?